elements/table-row.js

```diff
@@ -118,52 +118,44 @@ module.exports = Row
 function Row () {
   if (!(this instanceof Row)) return new Row()
   Nanocomponent.call(this)
+
+  this.parts = {
```

Comment:
`parts` is a bit generic, can we call this `partials` maybe? That term is known from the Ruby on Rails world